crates/common/mqtt_channel/src/connection.rs

```diff
@@ -235,4 +253,22 @@ impl Connection {
     pub(crate) async fn do_pause() {
         sleep(Duration::from_secs(1)).await;
     }
+
+    pub(crate) async fn subscribe_to_topics(
+        mqtt_client: &AsyncClient,
+        subscriptions: Vec<rumqttc::SubscribeFilter>,
+    ) -> bool {
+        if subscriptions.is_empty() {
+            eprintln!("ERROR: Subscriptions vector is empty");
+            return false;
```

Comment:
In no case it is an error to have no subscriptions !

Reply:
Does that mean that checking whether the topic list is empty or not is not necessary?